internal/prompt/prompt_js.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-2021 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package prompt
+
+import (
+	"bufio"
+	"fmt"
+
+	"github.com/btcsuite/btcwallet/internal/legacy/keystore"
+)
+
+func ProvideSeed() ([]byte, error) {
+	return nil, fmt.Errorf("prompt not supported in WebAssembly")
```

Comment:
Will these ever be invoked? Any reason to not panic instead?

Reply:
I'm guessing we don't need these directly as we don't plan to run the raw `btcwallet` binary (where these are used) directly in the WASM context.

Reply:
Yeah, we should never hit this code path. I personally don't really like calling `panic()` in general but I'm fine with either option as it shouldn't matter for our use case anyway.

Reply:
A panic would give us a clear indication that this was called when not expected, in contrast to an error which could go undetected, but I agree that it doesn't make much of a difference in this case.

Reply:
Good point. Though I found that panics in WASM aren't that easy to handle or find since the process just aborts and you don't even get a nice stack trace. So an error might make a tiny bit more sense in this specific context?